invenio_rdm_pure/config.py

```diff
@@ -93,3 +93,25 @@
 RDM_RECORDS_METADATA_NAMESPACES = {
     "tug": {"@context": "https://graz.pure.elsevier.com/"}
 }
+
+# Invenio-RDM-Pure
+# =========================
+#
+INVENIO_RDM_PURE_CONFIG_PRESENT = False
+## Invenio-RDM
+## ========================
+##
+RDM_HOST_URL = "https://127.0.0.1:5000/"
+RDM_RECORD_URL = RDM_HOST_URL + "api/records/{}"
+RDM_RECORDS_URL = RDM_HOST_URL + "api/records"
+RDM_PURE_USER_EMAIL = ""
+RDM_PURE_USER_PASSWORD = ""
+
+## Pure
+## ========================
+##
+PURE_API_URL = ""
+PURE_API_KEY = ""
+PURE_USERNAME = ""
+PURE_PASSWORD = ""
+PURE_RESPONSIBLE_EMAIL = ""
```

Comment:
```suggestion

# Pure
# ========================
#

PURE_API_URL = ""
"""What is it for - short comment needed."""

PURE_API_KEY = ""
"""What is it for - short comment needed."""

PURE_USERNAME = ""
"""What is it for - short comment needed."""

PURE_PASSWORD = ""
"""What is it for - short comment needed."""

PURE_RESPONSIBLE_EMAIL = ""
"""What is it for - short comment needed."""
```

Reply:
Also for configuration variables - it's better to use a prefix such as ```INVENIO``` or ```PURE``` to all newly added config vars.  open for discussion @utnapischtim .

Reply:
I added my comment to the issue #11 

In short, we should distinguish between between the configuration variables which belong to the repository and those which belong to the source/target system. 

For me calling the configuration variable PURE_PASSWORD makes it more clear that it is the password to connect the pure system. With INVENIO_PURE_PASSWORD it is not that clear for me.